besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -481,7 +482,7 @@ public Runner build() {
       miningCoordinator.addEthHashObserver(stratumServer.get());
     }
 
-    staticNodes.stream()
+    sanitizePeers(network, staticNodes)
```

Comment:
Do we have any tests covering static node injections?  To validate the peers were added?

Reply:
We don't. Will create a separate ticket